### PR TITLE
PLGMAG2V2-717: Add payment method icons

### PR DESCRIPTION
**Pull request by Vinod-MultiSafepay**

### CODE REVIEW

File: view/frontend/layout/hyva_checkout_components.xml [hunk 1]
```diff
@@ -29,6 +34,623 @@
...
+            <block name="checkout.payment.method.multisafepay_ideal"
+                   as="multisafepay_ideal">
+                <arguments>
+                    <argument name="metadata" xsi:type="array">
+                        <item name="icon" xsi:type="array">
+                            <item name="svg" xsi:type="string">multisafepay/multisafepay_ideal</item>
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
```
Comment (xgerhard) [inline view/frontend/layout/hyva_checkout_components.xml:259]: This is duplicate and will overwrite the iDeal config block at line 13. Therefor breaking the iDeal issuer selector.

**Comment by xgerhard** [comment]
Thanks for working on this feature! 
May I suggest adding a title attribute to the icons as well.

By default the method code is used as title:
msp_icon_1

Title example:
```html
<!-- Payment Renderer: MultiSafepay Mastercard -->
<block name="checkout.payment.method.multisafepay_mastercard"
       as="multisafepay_mastercard">
    <arguments>
        <argument name="metadata" xsi:type="array">
            <item name="icon" xsi:type="array">
                <item name="svg" xsi:type="string">multisafepay/multisafepay_mastercard</item>
                <item name="attributes" xsi:type="array">
                    <item name="title" xsi:type="string">Mastercard</item>
                </item>
            </item>
        </argument>
    </arguments>
</block>
```
Results in:
msp_icon_before

Some SVG's might already have a title/desc set in the .svg files, noticed this at the multisafepay_paypal.svg file, if those are set the config values are ignored.

File: view/frontend/layout/hyva_checkout_components.xml (hunk 1 above)
Comment (Vinod-MultiSafepay) [reply view/frontend/layout/hyva_checkout_components.xml]: Thanks!

**Comment by Vinod-MultiSafepay** [comment]
Thanks for the feedback @xgerhard , i've added the changes accordingly.

**Review by Daniel-MultiSafepay — APPROVED** [review]

**Pull request merged**